src/org/jetbrains/plugins/scala/lang/completion/statistician/ScalaCompletionStatistician.scala

```diff
@@ -11,6 +13,26 @@ import com.intellij.psi.statistics.StatisticsInfo
 
 class ScalaCompletionStatistician extends CompletionStatistician {
   def serialize(element: LookupElement, location: CompletionLocation): StatisticsInfo = {
-    StatisticsInfo.EMPTY //todo:
+    val currentElement = Option(element.as(LookupItem.CLASS_CONDITION_KEY)).getOrElse(return null)
+    ScalaLookupItem.original(currentElement) match {
+      case s: ScalaLookupItem if s.isLocalVariable => StatisticsInfo.EMPTY
+      case s: ScalaLookupItem => helper(s.element, location)
+      case _ => null //don't impact on java Lookups, no statistics for scala keyword elements
+    }
+  }
+
+  def helper(element: PsiNamedElement, location: CompletionLocation): StatisticsInfo = {
+    element match {
+      case member: PsiMember =>
+        val key = ScalaStatisticManager.memberKey(member).getOrElse(return StatisticsInfo.EMPTY)
+        val containingClass = member.getContainingClass
+        if (containingClass != null) {
+          val context = ScalaStatisticManager.memberKey(containingClass)
+            .getOrElse(return new StatisticsInfo("scalaMember#", key))
```

Comment:
It's better to do it without return, just getOrElse("scalaMember#"). One of the reasons is performance as return here in by-name-parameter position.